src/functionalTest/resources/application.yaml

```diff
@@ -36,7 +36,10 @@ idam:
       username: ${IDAM_USER_NAME:bulkscan+ccd@gmail.com}
       password: ${IDAM_USER_PASSWORD:Password12}
 
-document_management.url: http://dm-store:4460
+document_management:
+  url: http://dm-store:4460
+  documents:
+    url: http://dm-store:4460/documents
```

Comment:
Is there a reason we are replicating this ? we can reuse already defined url and append context and document uuid to it.

Reply:
I'll change it to use contextPath as you suggested below